spec/test/stored_queries/test_fulltext_search.py

```diff
@@ -55,6 +56,17 @@
     "Vaccinia virus WR 65-16",
     "Dengue virus 2 Jamaica/1409/1983",
     "Dengue virus 2 Thailand/NGS-C/1944",
+    # --- Escape chars (,:+-|"') ---
```

Comment:
are `(` and `)` also escape chars? Could you put them together and/or space-separate all the characters to be escaped, so it's clear that you're not using them as parentheses?